scripts/validate_docstrings.py

```diff
@@ -440,24 +463,28 @@ def validate_one(func_name):
         for param_err in param_errs:
             errs.append('\t{}'.format(param_err))
 
-    if not doc.returns:
-        errs.append('No returns section found')
+    if not doc.returns and "return" in doc.method_source:
+        errs.append('No Returns section found')
+    if "yield" in doc.method_source:
```

Comment:
This would ideally have the same structure as the returns check directly above it, but I think there's a bug in numpy doc where it doesn't parse Yields sections